backend/controllers/taskController.js

```diff
@@ -15,4 +18,98 @@ const createTask = asyncHandler(async (req, res) => {
   }
 });
 
-module.exports = { createTask };
+const getTasks = asyncHandler(async (req, res) => {
+  const header = req.headers["authentication"];
```

Comment:
For these headers, we should change this to `Authorization` where it's a support Header for auth.
(i.e. 
<img width="863" alt="image" src="https://user-images.githubusercontent.com/24628243/158245901-79e4c3e5-292d-4f71-a7a1-98ce04c13ef2.png">
rather than having our own header for `authentication`)

Reply:
The changes should also be made on any other subsequent files that happen to use "authentication" instead of `Authorization`

Reply:
Fixed: Changes are reflected in recent commits